### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Strflightmight09**

### CODE REVIEW

**Review by levyishai — CHANGES_REQUESTED** [review]
Mention the TRIGONLib

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+TRIGON 5990's robot code for the 2024 off-season robot, "PLACEHOLDER".
+
+# External Programs
+
+All the "Advantage Scope" related files (CAD glTF files, layouts, etc.) can be
+found [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)   
+Credit to team 6328 Mechanical Advantage, for "Advantage Kit" and "Advantage Scope".
+We use these a lot and highly recommend them!
+
+# TrigonLIB
```
Comment (levyishai) [inline README.md]: TRIGONLib

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+[![](https://jitpack.io/v/Programming-TRIGON/TRIGONLib.svg)](https://jitpack.io/#Programming-TRIGON/TRIGONLib)
```
Comment (levyishai) [inline README.md]: Place the Jitpack link in the the title

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+TRIGON 5990's library for hardware wrappers, utilities, commands, etc.
```
Comment (levyishai) [inline README.md]:
```suggestion

[TRIGON 5990's library](https://github.com/Programming-TRIGON/TRIGONLib) for automatic Advantage Kit and Simulation hardware wrappers, utilities, commands, and more.    
This library is used from the `org.trigon` import.
```

**Review by levyishai — CHANGES_REQUESTED** [review]

**Review by levyishai — COMMENTED** [review]
@Nummun14 feel free to leave a review as well

File: README.md (hunk 2 above)
Comment (Strflightmight09) [reply README.md]: Where exactly do you want me to put it?

Comment (levyishai) [reply README.md]: Next to the `# TRIGONLib `

**Review by levyishai — APPROVED** [review]

**Review by Nummun14 — APPROVED** [review]

**Pull request merged**